src/System.Net.Ping/src/System/Net/NetworkInformation/Ping.Windows.cs

```diff
@@ -31,14 +31,14 @@ public partial class Ping
 
         private PingReply SendPingCore(IPAddress address, byte[] buffer, int timeout, PingOptions options)
         {
-            // Since isAsync == false, DoSendPingCore will execute synchronously and return a completed 
+            // Since isAsync == false, DoSendPingCore will execute synchronously and return a completed
             // Task - so no blocking here
             return DoSendPingCore(address, buffer, timeout, options, isAsync: false).Result;
```

Comment:
Nit: it was preexisting your change, but this `.Result` should be changed to `.GetAwaiter().GetResult()`.  If the operation fails with an exception, `.Result` will end up wrapping the exception in an `AggregateException`, whereas `.GetAwaiter().GetResult()` will propagate the same exception instance (we want the latter).

Reply:
updated.